__tests__/api/chat/send-signed-message.test.js

```diff
@@ -63,4 +63,26 @@ describe('POST /chat/send-signed-message', () => {
       })
     );
   });
+
+  test('should return 403 error validation if channelType is not chat or group', async () => {
+    const response = await supertest(app)
+      .post('/api/v1/chat/send-signed-message')
+      .set({Authorization: `Bearer token`})
+      .send({
+        message: 'Hello World',
+        channelType: 3
+      })
+      .expect(403);
+
+    expect(response.body).toEqual(
+      expect.objectContaining({
+        message: 'Error validation',
+        error: expect.arrayContaining([
+          expect.objectContaining({
+            message: expect.any(String)
+          })
+        ])
+      })
+    );
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case added here is good for checking the validation of `channelType`. However, it would be better to also check the exact error message returned in the response. This will ensure that the correct error message is being sent when the `channelType` is not "chat" or "group". 

```diff
    expect(response.body).toEqual(
      expect.objectContaining({
        message: 'Error validation',
        error: expect.arrayContaining([
          expect.objectContaining({
-            message: expect.any(String)
+            message: 'Invalid channel type. Must be either "chat" or "group".'
          })
        ])
      })
    );
```
This change ensures that the specific error message is being checked, providing a more robust test.


<!-- This is an auto-generated comment by OSS CodeRabbit -->